lib/insights_cloud/async/insights_scheduled_sync.rb

```diff
@@ -23,7 +23,7 @@ def plan
       end
 
       def run
-        output[:status] = _('The scheduled process is disabled because this Foreman is configured with the use_local_advisor_engine option.') if ForemanRhCloud.with_local_advisor_engine?
+        output[:status] = _('The scheduled process is disabled because this Foreman is configured with a local Insights SmartProxy.') if ForemanRhCloud.with_insights_smartproxy?
```

Comment:
```suggestion
        output[:status] = _('The scheduled process is disabled because this Foreman is configured with a local Insights Smart Proxy.') if ForemanRhCloud.with_insights_smartproxy?
```
I think it should be two words